packages/components/src/local/rfi-status-board/index.tsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react'
 import { Badge, DataTable, RfiForm } from '@serge/components'
 import { MessageCustom } from '@serge/custom-types/message'
-import { CollaborativeMessageStates} from '@serge/config'
+import { CollaborativeMessageStates } from '@serge/config'
 import { ChannelData } from '@serge/custom-types'
 
-const RfiStatusBoard = ({ rfiData}: { rfiData: {rfiMessages:MessageCustom[], roles: string[], channels: Array<ChannelData>} }) => {
+/* Import Types */
+import Props from './types/props'
 
+/* Import Stylesheet */
+// import styles from './styles.module.scss'
```

Comment:
aren't we importing the styles?

Reply:
style will be imported in the same component but in next PR because we did more refactor on that PR #689